### PR TITLE
fix: text display entity data & teleport packet

### PR DESCRIPTION
**Pull request by GoodestEnglish**
The latest version of Fairy is using incorrect index of text display entity data, causing spawning a hologram will kick the player out from the sever. 
image

Also, the teleport packet in the text display hologram update function is using incorrect pitch and yaw, causing hologram looks into the wrong direction.  

This PR should fix both errors.

### CODE REVIEW

**Comment by LeeGodSRC** [comment]
wow goodest PR

File: framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java [hunk 1]
```diff
@@ -101,10 +101,10 @@ public void hide(MCPlayer player) {
     private List<EntityData> createEntityData(MCPlayer mcPlayer) {
         List<EntityData> entityDataList = new ArrayList<>();
 
-        entityDataList.add(new EntityData(22, EntityDataTypes.COMPONENT, MCAdventure.asItemString(line.render(mcPlayer), mcPlayer.getLocale()))); // text
-        entityDataList.add(new EntityData(23, EntityDataTypes.CAT_VARIANT, 200)); // line width
-        entityDataList.add(new EntityData(24, EntityDataTypes.CAT_VARIANT, 0x40000000)); // background color
-        entityDataList.add(new EntityData(25, EntityDataTypes.CAT_VARIANT, -1)); // text opacity
+        entityDataList.add(new EntityData(23, EntityDataTypes.COMPONENT, MCAdventure.asItemString(line.render(mcPlayer), mcPlayer.getLocale()))); // text
```
Comment (LeeGodSRC) [inline framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java]: Is the metadata index the same in lower versions? 1.19 and 1.20 for example.

Comment (GoodestEnglish) [reply framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java]: you are right, the index has been changed in 1.20.2. I will update it soon

Comment (GoodestEnglish) [reply framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java]: should be good now :)

Comment (LeeGodSRC) [reply framework/modules/mc/mc-hologram/src/main/java/io/fairyproject/mc/hologram/entity/impl/TextDisplayHologramEntity.java]: You are the goodest

**Review by LeeGodSRC — APPROVED** [review]
LGTM, goodest PR

**Pull request merged**